extensions/spark/kyuubi-spark-lineage/src/main/scala/org/apache/kyuubi/plugin/lineage/helper/SparkSQLLineageParseHelper.scala

```diff
@@ -301,6 +326,19 @@ trait LineageParser {
       case p: LocalRelation =>
         joinRelationColumnLineage(parentColumnsLineage, p.output, Seq(LOCAL_TABLE_IDENTIFIER))
 
+      case p: InMemoryRelation if p.cacheBuilder.tableName.nonEmpty =>
```

Comment:
now we support cached plan which does not have table name

Reply:
Can you give some real cache table cases without table name ? So that I can test it